src/transformers/trainer.py

```diff
@@ -432,6 +432,10 @@ def __init__(
                 self.fsdp = ShardingStrategy.SHARD_GRAD_OP
             elif FSDPOption.NO_SHARD in args.fsdp:
                 self.fsdp = ShardingStrategy.NO_SHARD
+            elif FSDPOption.HYBRID_SHARD in args.fsdp:
```

Comment:
These are no longer used with PyTorch. They are only used with FSDP XLA integration. For PyTorch, it uses Accelerate's integration via the codeblock here: 

https://github.com/huggingface/transformers/blob/38e2633f80a4924bf613b0240622492beee4cfcc/src/transformers/training_args.py#L1643-L1673

Reply:
ok. I see, thx.